### PR TITLE
Graph ql setup

### PR DESCRIPTION
**Pull request by yafet-dev**
//test

### CODE REVIEW

**Review by yonas-seyoum — APPROVED** [review]

File: services/service-core/package.json [hunk 1]
```diff
@@ -21,8 +21,13 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
+    "@apollo/server": "^4.9.0",
+    "@as-integrations/fastify": "^2.1.0",
+    "@mercuriusjs/gateway": "^2.0.0",
```
Comment (peterrogov) [inline services/service-core/package.json:26]: what is this? is this needed?

File: services/service-core/src/graphql/users/test/users.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  let usersService: UsersService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [UsersService]
+    }).compile();
+    usersService = module.get<UsersService>(UsersService);
+  });
+
+  it('', () => {
```
Comment (peterrogov) [inline services/service-core/src/graphql/users/test/users.service.spec.ts:26]: Test must have a descriptive name. Something like `it('must have userService defined')`

File: services/service-core/src/graphql/users/test/users.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    expect(usersService).toBeDefined();
```
Comment (peterrogov) [inline services/service-core/src/graphql/users/test/users.service.spec.ts:27]: In a nutshell, what is the purpose of this test? What does it test?

File: services/service-core/src/graphql/users/test/users.resolver.spec.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [UsersResolver]
+    }).compile();
+    usersResolver = module.get<UsersResolver>(UsersResolver);
+  });
+
+  describe('findAll', () => {
+    it('', () => {
```
Comment (peterrogov) [inline services/service-core/src/graphql/users/test/users.resolver.spec.ts:27]: Please add a descriptive name to the test

**Review by peterrogov — APPROVED** [review]
Approved. But there are a few comments that need to be fixed. Please see if you can fix this before merging

**Pull request merged**